src/ert/config/queue_config.py

```diff
@@ -139,17 +120,166 @@ def generate_dict(option_list: List[Tuple[str, str]]) -> Dict[str, List[str]]:
             )
 
 
-def _validate_torque_options(torque_options: List[Tuple[str, str]]) -> None:
-    for option_strings in torque_options:
+queue_memory_options: Mapping[str, List[str]] = {
+    "LSF": [],
```

Comment:
Is it more consistent to use `QueueSystem.LSF` instead of the string `LSF`?

Reply:
I had that before, but I changed it due to know these `*_options` are exposed in tests and didn't want to import all the Cwrap classes there.

Reply:
ok!